### ISSUE TITLE
Export Report doesnt have Notes

### ISSUE THREAD

**Issue by pamvee** [MicrosoftDocs/microsoft-365-docs#9521]
The statement reads "The identifiable user data associated with improvement actions includes any implementation and testing work done, documents they uploaded, and any notes they entered." under "Manage User History" section.  However, the report we get doesnt actually include details of work done and no notes exist as part of the export. The export report contains all IAs that have been assigned to the user at any point and the statuses of them at the time of report generation. It does however contain names of documents that the user has uploaded. Here is a sample:
![image](https://user-images.githubusercontent.com/110828209/186657803-653aa3c1-f55b-41c9-9912-e221b7d55585.png)

It is suggested to reword this to better reflect this. Thank you!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 69f424cd-302d-5845-34fb-7b906bf7970c
* Version Independent ID: cfa50ebe-7ef1-de51-4456-a1212be9e7ec
* Content: [Get started with Microsoft Purview Compliance Manager - Microsoft Purview (compliance)](https://docs.microsoft.com/en-us/microsoft-365/compliance/compliance-manager-setup?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/compliance-manager-setup.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/compliance-manager-setup.md)
* Service: **o365-seccomp**
* GitHub Login: @chvukosw
* Microsoft Alias: **chvukosw**

**Comment by msbemba** [comment]
@pamvee Thank you for your feedback. We have made changes to the document via PR#https://github.com/MicrosoftDocs/microsoft-365-docs/pull/9535

Once the author approves, the changes will be live.

Thanks
Sri

**Comment by chvukosw** [comment]
Thanks for the updated info; the PR has been merged.
